pubsub/integration_test.go

```diff
@@ -2011,3 +2014,49 @@ func TestExactlyOnceDelivery_PublishReceive(t *testing.T) {
 	// Tests for large messages (larger than the 4MB gRPC limit).
 	testPublishAndReceive(t, client, 0, false, true, 1, 5*1024*1024)
 }
+
+func TestIntegration_TopicUpdateSchema(t *testing.T) {
+	ctx := context.Background()
+	// TODO(hongalex): update these staging endpoints after schema evolution is GA.
+	c := integrationTestClient(ctx, t, option.WithEndpoint("staging-pubsub.sandbox.googleapis.com:443"))
```

Comment:
Let's just not even merge until we can remove this. That should hopefully be next week.